packages/fast-components-react-base/package.json

```diff
@@ -103,6 +104,7 @@
     "@microsoft/fast-web-utilities": "^1.9.2",
     "exenv-es6": "^1.0.0",
     "lodash-es": "^4.0.0",
+    "raf-polyfill": "^1.0.0",
```

Comment:
I don't think we need `raf-polyfill` as a peer dependency - just a dev dependency because our shipped code doesn't actually reference it.